recipes/rocksdb/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "rocksdb" %}
+{% set version = "6.2.2" %}
+
+package:
+  name: rocksdb
+  version: {{ version }}
+
+source:
+  url: https://github.com/facebook/rocksdb/archive/v{{ version }}.tar.gz
+  sha256: 3e7365cb2a35982e95e5e5dd0b3352dc78573193dafca02788572318c38483fb
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler("cxx") }}
+    - cmake >=3.5.1,<4
+
+  host:
+    - gflags >=2.2.2,<3
+    - jemalloc >=4.5.0,<5
+    - snappy >=1.1.7,<2
```

Comment:
I might be missing something, but shouldn't we be depending on the pinning provided by https://github.com/conda-forge/conda-forge-pinning-feedstock/ for host deps? (Unless you want to override that for a reason)

Reply:
^ yes! We need to add a hint on this to the linter.

Reply:
Two questions:
  - What should we do when a requirement is not in conda-forge-pinning, e.g. `gflags`, and `jemalloc`?
  - What to do when our requirement conflicts or is stricter than conda-forge-pinning? Not our  here but I am wondering...

Reply:
If it's not in the pinning, add it with the latest version and also add a migration to ensure that al packages using it are pulled to using that version.

Reply:
We might consider checking the graph to see what packages are currently listed in host but are not pinned. Not that they *should* be pinned automatically, just that we are aware and make a conscious decision to not pin those packages.

@sodre in the case where the pins are more strict than conda-forge or conflict, this is mostly ok. Most likely this will cause environments using these packages to use older code. These packages will be left behind by migrations though, so they won't be able to be in the same env as newer pins.

Reply:
@CJ-Wright, any issues on leaving the versions of `jemalloc` and `gflags` in the recipe until we decide if pinning is necessary?

Reply:
I don't have a problem with it, although your pins are most likely too strict based on the run_exports those packages provide.

Reply:
How many other packages depend on those packages?

Reply:
do you have a link to a graph? I tried github search on the org. The results did not look correct.

Reply:
The retro cf-scripts repo has some docs on how to interact with the graph, you might also look at conda-forge by_the_numbers.